src/views/partials/board-settings.handlebars

```diff
@@ -0,0 +1,57 @@
+
+<!-- Css for dropdown menu on w3schools -->
+<div class="dropdown">
```

Comment:
Why is there this dropdown?

Reply:
The dropdown is because you wanted me to merge the invite, and board settings into one, so I put it into a menu

Reply:
You can just put it all on the screen at once